pydantic/config.py

```diff
@@ -25,6 +25,7 @@
 Extra = _Extra()
 
 ExtraValues = Literal['allow', 'ignore', 'forbid']
+JsonType = Dict[str, Union[str, int, float, bool, None, List[Any], Dict[str, Any]]]
```

Comment:
`JsonType` is not the same as [Samuel suggestion](https://github.com/pydantic/pydantic/issues/6348#issuecomment-1618135577) on the issue.

Reply:
checked once again. I added the types as mentioned in the issue.